nengo/neurons.py

```diff
@@ -241,11 +245,19 @@ class RectifiedLinear(NeuronType):
 
     probeable = ('rates',)
 
+    amplitude = NumberParam('amplitude', low=0, low_open=True)
```

Comment:
I would allow `amplitude=0`. There might be some legitimate, although rare, use cases (e.g., I want to disable the output of an ensemble for testing purposes).

Reply:
Note that this addition was made to be consistent with `RectifiedLinear`.

Reply:
More importantly, though, I believe (?) if you try this you will get a decoder solver error due to an all-zero activity matrix.

Reply:
Ah yeah that's true.  Would still be nice to support, but not so nice that it's worth fiddling with all that 😄

Reply:
`LIFRate` also has an amplitude parameter, so I just set one up to be the same.